widgets/Icons/index.tsx

```diff
@@ -152,7 +154,7 @@ const IconSet = {
   ITEM_MOVE_UP: toSvg(ITEM_MOVE_UP),
   OUTLINER: toSvg(OUTLINER),
   LOGO: toSvg(LOGO),
-  LOGO_FC: toSvg(LOGO_FC),
+  LOGO_FC: toSvg(LOGO_FC, true),
```

Comment:
:/

Maybe we should revise how we import svgs?

Reply:
meh, this approach is fine imo. But if this condition here is too ugly for your liking we could also just import this specific one directly from where it is used, without wrapping it in `toSVG` so it doesn't get the `custom-icon` class.
The problem is that once you set `fill` to something in CSS for an SVG, there is no way to reset it to the actual color of the SVG, so you need to avoid setting it in one way or another

Reply:
I'm also talking about the fact that we're creating a very big object which makes tree shaking impossible. However, cleaning up might be enough.

We could just package the actual svg now that I think about. Let me try.